statistics-dashboard-service/src/main/java/eu/europeana/statistics/dashboard/service/persistence/MongoSDDao.java

```diff
@@ -168,4 +170,31 @@ public List<Target> getCountryTargets(){
     result.forEach(queryResult::add);
     return queryResult;
   }
+
+  /**
+   * Create a Historical object (with the current time) for the given country
+   */
+  public Historical generateHistoricalSnapshot(String country){
```

Comment:
Could you change the name of the method to something else?
`generateLatestTargetData` or something on those lines would be better I think.

Reply:
Done